doc/cluster-management/byo.md

```diff
@@ -24,8 +24,8 @@ It is necessary to have access to the following infrastructure:
 * A means to serve the OCK OSTree archive
 * A means to serve Ignition files
 
-An easy way to achieve the first four bullet points is to download [https://yum.oracle.com/oracle-linux-isos.html](Oracle Linux installation media)
-and following the [https://docs.oracle.com/en/operating-systems/oracle-linux/8/install/install-AutomatinganOracleLinuxInstallationbyUsingKickstart.html](automated installation guide).
+An easy way to achieve the first four bullet points is to download [https://yum.oracle.com/oracle-linux-isos.html] (Oracle Linux installation media)
```

Comment:
The issue is that the link and text are reversed.  See https://www.markdownguide.org/basic-syntax/#links

Reply:
understood